vips/vip-416/bsctestnet.ts

```diff
@@ -0,0 +1,234 @@
+import { NETWORK_ADDRESSES } from "src/networkAddresses";
+import { LzChainId, ProposalType } from "src/types";
+import { makeProposal } from "src/utils";
+
+import { COMPTROLLERS as OPBNBTESTNET_COMPTROLLERS } from "../../multisig/proposals/opbnbtestnet/vip-024";
+import { VTOKENS as OPBNBTESTNET_VTOKENS } from "../../multisig/proposals/opbnbtestnet/vip-024";
+import { POOL_REGISTRY as OPBNBTESTNET_POOL_REGISTRY } from "../../multisig/proposals/opbnbtestnet/vip-024";
+import { NTGs as OPBNBTESTNET_NTGs } from "../../multisig/proposals/opbnbtestnet/vip-024";
+import { PSR as OPBNBTESTNET_PSR } from "../../multisig/proposals/opbnbtestnet/vip-024";
+import { PLP as OPSEPOLIA_PLP, PRIME as OPSEPOLIA_PRIME } from "../../multisig/proposals/opsepolia/vip-008";
+import { COMPTROLLERS as OPSEPOLIA_COMPTROLLERS } from "../../multisig/proposals/opsepolia/vip-008";
+import { VTOKENS as OPSEPOLIA_VTOKENS } from "../../multisig/proposals/opsepolia/vip-008";
+import { POOL_REGISTRY as OPSEPOLIA_POOL_REGISTRY } from "../../multisig/proposals/opsepolia/vip-008";
+import { NTGs as OPSEPOLIA_NTGs } from "../../multisig/proposals/opsepolia/vip-008";
+import { PSR as OPSEPOLIA_PSR } from "../../multisig/proposals/opsepolia/vip-008";
+import { REWARD_DISTRIBUTORS as OPSEPOLIA_REWARD_DISTRIBUTORS } from "../../multisig/proposals/opsepolia/vip-008";
+
+export const OPSEPOLIA_XVS_STORE = "0xE888FA54b32BfaD3cE0e3C7D566EFe809a6A0143";
+export const OPBNBTESTNET_XVS_STORE = "0x06473fB3f7bF11e2E8EfEcC95aC55ABEFCb2e0A0";
+
+export const OPBNBTESTNET_XVS_BRIDGE_ADMIN = "0x19252AFD0B2F539C400aEab7d460CBFbf74c17ff";
+export const OPSEPOLIA_XVS_BRIDGE_ADMIN = "0x6bBcB95eCF9BEc9AE91d5Ad227783e3913145321";
+
+export const OPSEPOLIA_BOUND_VALIDATOR = "0x482469F1DA6Ec736cacF6361Ec41621f811A6800";
+export const OPBNBTESTNET_BOUND_VALIDATOR = "0x049537Bb065e6253e9D8D08B45Bf6b753657A746";
+
+const { opsepolia, opbnbtestnet } = NETWORK_ADDRESSES;
+
+const vip418 = () => {
+  const meta = {
+    version: "v2",
+    title: "VIP-332 accept ownership & give permissions to Normal Timelock",
```

Comment:
Update the titles of the VIP's. We can remove the VIP number, and I would add some more specific, like `Accept ownership/admin of contracts on OP sepolia and opBNB testnet`